pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -94,39 +94,42 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 	// Fetch the BuildRun instance
 	buildRun := &buildv1alpha1.BuildRun{}
 	err := r.client.Get(context.TODO(), request.NamespacedName, buildRun)
-	if err != nil && !errors.IsNotFound(err) {
-		reqLogger.Error(err, "Failed to get the build instance")
+	if err != nil && !apierrors.IsNotFound(err) {
```

Comment:
same as before, the usage of apierrors seems not needed, but I might be missing something, can this just be?:

```
if err := r.client.Get(context.Background(), request.NamespacedName, buildRun); err != nil{
		return reconcile.Result{}, err
}
```

Reply:
See above discssion ^^